schema/base/land_cover.yaml

```diff
@@ -0,0 +1,50 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: Land Cover Schema
```

Comment:
This should be just `land_cover`.

(We changed the way all the feature titles are done...)